apps/demo/src/app/app.module.ts

```diff
@@ -38,16 +39,17 @@ import { DemoCartRootModule } from './cart/cart-root.module';
 
     HttpClientModule,
 
-    InMemoryModule,
-    
+    // InMemoryModule,
```

Comment:
I don't think we should make any changes in this file.

Reply:
I thought it was necessary to break the demo but I was wrong. Changed in [`284a5d1`](https://github.com/graycoreio/daffodil/pull/691/commits/284a5d18ce9adaf28c04a15a92ff7d9cd9556ee9).